vignettes/MetaboCoreUtils.Rmd

```diff
@@ -546,6 +546,100 @@ Generally, injecting study samples in random order can reduce (or even avoid)
 influence of any related technical bias in the downstream analysis and is highly
 suggested to improve and assure data quality.
 
+## Filtering data: Identifying measurement error 
```

Comment:
Maybe better use here a title `## Basic quality assessment and pre-filtering of metabolomics data`